AuthenticatorShared/Core/Vault/Models/Domain/AuthenticatorItem.swift

```diff
@@ -1,3 +1,4 @@
+import AuthenticatorBridgeKit
```

Comment:
seems like this isn't necessary

Reply:
Ah, good catch. The earlier version had an init that took in `AuthenticatorBridgeItemDataView`, but I removed that in the later commits. 👍